.github/workflows/db-migrations.yml

```diff
@@ -0,0 +1,38 @@
+name: Database Migrations
+
+on:
+  push:
+    paths:
+    - .github/workflows/db-migrations.yml
+    - lib/model/migrations/**
+    - test/db-migrations/**
+
+jobs:
+  standard-tests:
+    timeout-minutes: 2
+    # TODO should we use the same container as circle & central?
```

Comment:
Any downside to using the same container?

Reply:
This comment is duplicated from other github actions config files.